pagopa/pn-frontend
# Feat/pn 3820 aggiustamento testi timeline

## Short description
Verify how the labels, tooltips, descriptions and legal fact link labels appear in the timeline, for **all**  states and events in both digital and analog flows, including the distinction between single- and multi-recipient notification in many cases. 

## List of changes proposed in this pull request
- adaptation of labels, tooltips and description for single- / multi-recipient cases.
- deletion of separated the indication about the recipient that was present in events for multi-recipient notifications, since the distinction is made inside the descriptions.
- inclusion of city and ZIP code in the address for events related to the analog flow.

## How to test
With a **lot** of patience. Several different cases should be considered, and for each one, check every label, tooltip and description appearing in the timeline. You can try in either PA, PF and PG webapps, but in any case, at least a quick tour on *each* of them should be performed.
